myfitnesspal/entry.py

```diff
@@ -1,11 +1,22 @@
 from myfitnesspal.base import MFPBase
+import re
 
 
 class Entry(MFPBase):
     def __init__(self, name, nutrition):
         self._name = name
         self._nutrition = nutrition
 
+
+    #split out quantity and measuring unit out of entry name
+        regex = r'(?P<short_name>.+), (?P<quantity>\d+.*) (?P<unit>\w+|serving\(s\))(?: \(.+\))?'
+        match = re.search(regex, name)
+
+        self._quantity = match.group('quantity')
```

Comment:
If, for some reason, MyFitnessPal changes how they display the above values enough that your regex no longer matches, these `match.group` calls will fail by raising an `AttributeError` because the returned `match` above will be `None`.  Could I convince you into changing this into something like this?

```python
self._quantity = None
self._unit = None
self._short_name = None
if match:
    self._quantity = match.group('quantity')
    self._unit = match.group('unit')
    self._short_name = match.group('short_name')
```

That way, if there are changes, things won't be _broken_, they'll just be missing information.

(Note that I didn't test the above, so there may be typos)